apps/web/components/LanguageSelector/LanguageSelector.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="w-full bg-white flex items-center justify-center flex-col shadow-xl md:hidden">
+    <SfButton
+      v-for="locale in localeObj"
+      :key="locale"
+      variant="tertiary"
+      square
+      :aria-label="$t('lang.' + locale.code?.toString())"
+      class="py-3 cursor-pointer"
+      @click="switchLocale(locale.code?.toString())"
+      >{{ $t('lang.' + locale.code.toString()) }}</SfButton
+    >
+  </div>
+  <div class="hidden md:flex w-full bg-white items-center justify-center absolute z-[99999] top-[10%] shadow-xl">
+    <SfButton
+      v-for="locale in localeObj"
+      :key="locale"
+      variant="tertiary"
+      square
+      :aria-label="$t('lang.' + locale.code?.toString())"
+      class="px-10 py-10 cursor-pointer"
```

Comment:
i know it was not specified in task ,but should't the selected language use some bold font ?

Reply:
I don t see it as bold in the mockups